main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
```

Comment:
Consider using a function for reusability. Also, since the condition for the while loop is not multi-line, maybe we can drop the parentheses.